solutions.md

```diff
@@ -28,6 +28,7 @@ description: Applications & Solutions with Signal K Support
 - [WilhelmSK](https://itunes.apple.com/us/app/wilhelmsk/id1150499484?mt=8) : Highly customizable Signal K application for iOS with Apple Watch support
 - [SailRacer.net](http://sailracer.net) : Tactical app for iOS and Android - helps to make decisions faster. Polars, Charts, Wind graphs, Signal K, Google wear and much more.
 - [SignalK Monitor](https://www.youtube.com/watch?v=X8VbkD8WYV8): Intuitive and easy-to-use [Android](https://play.google.com/store/apps/details?id=com.belmille.signalkflutter) and [iPhone](https://apps.apple.com/fi/app/signalk-monitor/id1534189860) application to visualize SignalK data.
+- [SKipper](https://www.skipper.net/) : Highly customaizable Signal K application for Linux, Android, IOS, Windows and web!
```

Comment:
Maybe you want to correct the customizable typo?

Reply:
Yes, it has been long day! Will fix it soon!